.github/workflows/test.yml

```
@@ -196,12 +196,16 @@ jobs:
       group:  phoenix
       labels: ${{ matrix.runner }}
     env:
-      NODE_OPTIONS: ${{ matrix.cluster == 'phoenix' && '--max-old-space-size=2048' || '' }}
       ACTIONS_RUNNER_FORCE_ACTIONS_NODE_VERSION: node16
       ACTIONS_ALLOW_USE_UNSECURE_NODE_VERSION: true
     steps:
+      - name: Clean workspace
+        run: rm -rf "$GITHUB_WORKSPACE"/* "$GITHUB_WORKSPACE"/.[!.]* 2>/dev/null || true
```

actionlint (1.7.10)

[error] shellcheck reported issue in this script: SC2115:warning:1:8: Use "${var:?}" to ensure this never expands to /* 

(shellcheck)